docker/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
       - "80"
       - "8001"
     ports:
-      - "8080:80"
+      - "8088:80"
```

Comment:
😕 ❓

Reply:
It was conflicting with port `8080` for the app